src/edge/edgeManager.ts

```diff
@@ -174,6 +178,54 @@ export class EdgeManager {
         }
     }
 
+    public async selectDefaultPlatform(outputChannel: vscode.OutputChannel) {
+        const platforms: Platform[] = Platform.getPlatformsSetting();
+        const keyWithAlias: string[] = [];
+        const defaultKeys: string[] = [];
+        const platformMap: Map<string, any> = new Map();
+        platforms.forEach((value: Platform) => {
+            const displayName: string = value.getDisplayName();
+            if (!value.alias) {
+                defaultKeys.push(displayName);
+            } else {
+                keyWithAlias.push(displayName);
+            }
+            platformMap.set(displayName, value);
+        });
+        const platformNames: string[] = (keyWithAlias.sort()).concat(defaultKeys.sort());
+        const defaultPlatform = await vscode.window.showQuickPick(platformNames, { placeHolder: Constants.selectPlatform, ignoreFocusOut: true });

```

Comment:
UX nit: `ignoreFocusOut: true` here does not provide too much value, and when users trigger the quick picks via mouse-clicking the status bar item and would like to dismiss, they must reach the `ESC` key on keyboard. This is inconvenient.

Reply:
Do we have guide lines to use ignoreFocusOut? This could avoid the clicking mouse outside focus and cause command exit.